src/libraries/System.Memory/tests/MemoryPool/MemoryPool.cs

```diff
@@ -231,15 +231,15 @@ public static void MemoryPoolTryGetArray()
         [Fact]
         public static void ExtraDisposesAreIgnored()
         {
-            IMemoryOwner<int> block = MemoryPool<int>.Shared.Rent(42);
+            using IMemoryOwner<int> block = MemoryPool<int>.Shared.Rent(42);
```

Comment:
This using should not be necessary.  This test is explicitly validating Dispose, and it's calling it twice already.